index.bs

```diff
@@ -195,11 +212,32 @@ When an <a>image request</a> has become <a>completely available</a> and decoded,
 
 <div algorithm="image element loaded">
     1. Let |image request| be the input <a>image request</a>.
-    1. If |image request|'s <a>associated element</a> is not <{img}>, <{image}>, or <a>background-image</a>, abort these steps.
-    1. If the <a>timing allow check</a> passes for |image request|'s resource, then add |image request| to <a>imagesPendingRendering</a>.
-    1. Otherwise, run the <a>report element timing</a> algorithm, passing in |image request| and 0.
+    1. Let |element| be |image request|'s <a>associated element</a>.
```

Comment:
Unrelated to this PR, but seems like the "associated element" concept is not so well defined...

Reply:
Yep :( I'm not sure how to fix this though, I'll need some help to come up with a sound definition for this.

Reply:
Yeah, let's take that to another issue